src/MSBuild.Sdk.SqlProj.Templates/templates/sqlproj/.template.config/template.json

```diff
@@ -0,0 +1,23 @@
+{
+    "$schema": "http://json.schemastore.org/template",
+    "author": "MSBuild.Sdk.SqlProj",
+    "classifications": [ "Database", "SqlServer" ],
+    "identity": "MSBuild.Sdk.SqlProj",
+    "name": "SQL Server Database Project",
+    "shortName": "sqlproj",
```

Comment:
Are we competing / colliding with Microsoft here - or are we just first movers?

Reply:
That's a good question. I don't necessarily want to compete, I'm just trying to fill a gap with this project. I made a screenshot of how this shows up in Visual Studio currently. As you can see it is clearly marked with the MSBuild.Sdk.SqlProj postfix, so at least you can distinguish between the old and new projects. Its also tagged with Linux and macOS so that you can see you can use this project cross-platform.

![Screenshot 2020-08-31 at 09 44 51](https://user-images.githubusercontent.com/1225489/91695559-a37c8c80-eb6e-11ea-8c00-b05a26cb086b.png)

Do you think it is going to be a problem if we use this name?

Reply:
No, I doubt it will be any issue.. 

We are doing MS a favor here, and I have only heard positive feedback from the product team.

Maybe add description metadata to the "template.json", 
"A project that builds and publishes a SQL Server Data-Tier Application package (.dacpac)"

(Corresponding to "A project for creating a SQL Server database.")

Reply:
Fair enough. I've added the Description to the template now.